app/views/claims/_ineligibility_reason_ineligible_claim_school.html.erb

```diff
@@ -10,6 +10,9 @@
 
 <p class="govuk-body">
   If you taught at more than one school during this period, you can
-  <%= link_to "start your claim again", new_claim_path, class: "govuk-link" %> with another school.
+  search again with the next school.
   You only need to have worked at one eligible school to claim.
 </p>
+
+<%= link_to "Enter another school", claim_path(policy: current_policy_routing_name, slug: "claim-school", additional_school: true), class: "govuk-button" %>
+<%= link_to "I've tried all of my schools", claim_path(policy: current_policy_routing_name, slug: "ineligible", tried_all_schools: true), class: "govuk-button govuk-button--secondary" %>
```

Comment:
As with the template, if we use a more generic name than `:tried_all_schools` then we make the code for displaying a generic ineligible message more re-useable.